apps/condo/pages/employee/[id]/index.tsx

```diff
@@ -282,39 +308,51 @@ export const EmployeeInfoPage: PageComponentType = () => {
     const { query } = useRouter()
     const { link } = useOrganization()
     const intl = useIntl()
-    const UpdateEmployeeMessage = intl.formatMessage({ id: 'employee.UpdateTitle' })
+    const LoadingInProgressMessage = intl.formatMessage({ id: 'LoadingInProgress' })
     const ErrorMessage = intl.formatMessage({ id: 'errors.LoadingError' })
+    const NotFoundMsg = intl.formatMessage({ id: 'NotFound' })
 
     const employeeId = String(get(query, 'id', ''))
+
     const { obj: employee, loading, error, refetch } = OrganizationEmployee.useObject(
         {
             where: {
                 id: employeeId,
             },
         }
     )
+
     const { objs: organizationEmployeeSpecializations } = OrganizationEmployeeSpecialization.useObjects({
         where: {
             employee: { id: employeeId },
         },
     })
 
+    const { data: activeTicketsOrganizationEmployeeCount } = useGetOrganizationEmployeeTicketsCountForReassignQuery({
+        variables: {
+            userId: employee?.user?.id,
```

Comment:
this means that if userId is undefined for some reason, we will fire a gql request and will get 400 bad request error